### PR TITLE
[Docs] Update custom documentation for loss layer

### PR DESCRIPTION
**Pull request by zhoonit**
- [Docs] Update custom documentation for loss layer

```
This patch updates documentation of custom for loss layer.
Also deletes redundant `pow.cpp` that sneaked in.

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: Jihoon Lee <jhoon.it.lee@samsung.com>

```

### CODE REVIEW

**Review by lhs8928 — APPROVED** [review]
LGTM

File: Applications/Custom/README.md [hunk 1]
```diff
@@ -4,16 +4,22 @@ Applications inside this folder are dedicated to demonstrate how to create custo
...
 
-`*Plugin` demonstrates how to generate an object inside a plugin code and register on the fly.
-For example, `OptimizerPlugin` will demo about how to create a custom optimizer as a pluggable library.
+1. `LayerClients/` has a demo about how to write a custom layer and register it.
+2. `LayerPlugin/` contains test and build scripts to show how to build a custom layer and object as a seperate library(`.so`) and register it.
+
+#### Related Custom Objects:
+
+
+1. **Pow Layer**: A custom layer object which get x -> returns x^(exponent), exponent can be set by `layer.setProperty({"exponent={n}"});`.
```
Comment (kparichay) [inline Applications/Custom/README.md:24]: is it ok to delete pow layer and keep description?

Comment (zhoonit) [reply Applications/Custom/README.md:24]: The deleted pow layer is duplicated one, that sneaked in 🤣

Comment (kparichay) [reply Applications/Custom/README.md:24]: haha ok.

**Review by kparichay — APPROVED** [review]
LGTM

**Review by jijoongmoon — APPROVED** [review]
LGTM

**Pull request merged**